governance/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unlock-protocol/governance",
-  "packageManager": "yarn@3.5.1",
+  "packageManager": "yarn@4.0.1",
```

Comment:
Do we want to put that at the root of our repo as well?

Reply:
Oh it's there I think, so any reason why it needs to be here too?

Reply:
let's delete them